### DISCUSSION TITLE
Compile error on September 2 commit on mbyte.c

### DISCUSSION THREAD

**Discussion by surmish** [neovim/neovim#30346] [Q&A] [answered]
I see this error when I run `make`.

```
neovim/src/nvim/mbyte.c:482:29: error: ‘utf8proc_property_t’ {aka ‘const struct utf8proc_property_struct’} has no member named ‘ambiguous_width’
  482 |   if (*p_ambw == 'd' && prop->ambiguous_width) {
```

Anyone else seeing this?

**Comment by clason** [comment]
The `utf8proc` dependency was bumped and needs to be rebuilt. Always run `make distclean` in case of any build errors.

**Comment by gabrielfrimodig** [comment, marked as answer]
This error occurs most likely because your `utf8proc` library is outdated and lacks the `ambiguous_width` member that neovim expects. Try updating `utf8proc` to the latest version and let me know if that resolves the issue.

**Comment by surmish** [comment]
Thanks I cloned this repo `https://github.com/JuliaStrings/utf8proc` and compiled it. That resolved the bug.
